src/Orc.SupportPackage.Xaml/Views/SupportPackageWindow.xaml.cs

```diff
@@ -36,6 +37,8 @@ public SupportPackageWindow(SupportPackageViewModel viewModel)
             : base(viewModel, DataWindowMode.Custom)
         {
             InitializeComponent();
+            
+            CloseWindowButton.Command = new Command(ExecuteClose, OnCloseCanExecute);
```

Comment:
Why not using AddCustomButton?

Reply:
Can i style it somehow? I want imy close button to be the same as other action button in these window. Or it doesn't matter, what do you think?